images-styles.html

```diff
@@ -0,0 +1,45 @@
+            <figcaption><strong>YOUTUBE</strong></figcaption>
+        </figure>
+        <figure>
+            <img src="./flipkart.jpeg" alt="img not found"/>
+            <figcaption><strong>FLIPKART</strong></figcaption>
+        </figure>
```

Comment:
Requirement was a list of images with caption.  Semantically that would mean that you embed the figure tags inside ul>li tag.